packages/mongodb-chatbot-ui/src/services/conversations.ts

```diff
@@ -62,12 +62,25 @@ export class TimeoutError<Data extends object = object> extends Error {
   }
 }
 
+/**
+  Options to include with every fetch request made by the ConversationService.
+  This can be used to set headers, etc.
+ */
+export type ConversationFetchOptions = Omit<
+  RequestInit,
+  "body" | "method" | "headers" | "signal"
+> & {
+  headers?: Record<string, string>;
```

Comment:
Should we accept a standard [Headers](https://developer.mozilla.org/en-US/docs/Web/API/Headers) object here?

I'm personally partial to using web standards when possible. In this case it can guarantee that headers are well-formed and in general is less for us to document and maintain.

Reply:
there was some reason why i didn't..don't recall what it was..probably not a great reason. will reapproach

Reply:
took a bit of typescript-foo to use the headers object but got it working